src/react-components/src/components/03-organisms/Hero/Hero.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from "react";
+
+import Image from "../../01-atoms/Images/Image/Image";
+import Heading from "../../01-atoms/Text/Headings/Heading";
+import BodyText from "../../01-atoms/Text/Text/BodyText";
+import { HeroTypes } from "./HeroTypes";
+import Hero from "./Hero";
+import SectionName from "../../00-base/SectionName";
+import Placeholder from "../../00-base/Placeholder";
+
+export default {
+  title: "Hero",
+  component: Hero,
+};
+
+export const heroPrimary = () => <Hero
+  heroType={HeroTypes.Primary}
+  heading={<Heading
+    level={1}
+    id={"1"}
+    text={"Hero Primary"}
+    blockName={"hero"}
+  />}
+  backgroundImageSrc="https://placeimg.com/1600/800/arch"
+/>;
+
+export const heroSecondary = () => <Hero
+  heroType={HeroTypes.Secondary}
+  heading={<Heading
+    level={1}
+    id={"1"}
+    text={"Hero Secondary"}
+    blockName={"hero"}
+  />}
+  subHeaderText={<BodyText blockName="hero">Example subtitle</BodyText>}
+  image={<Image
+    src="https://placeimg.com/800/400/arch"
+    isDecorative={true}
+    imageBlockname={"hero"}
+  />}
+/>;
+
+export const heroStaffPicks = () => <Hero
+  heroType={HeroTypes.Primary}
+  heading={<Heading
+    level={1}
+    id={"1"}
+    text={"Staff Picks"}
+    blockName={"hero"}
+  />}
+  subHeaderText={<BodyText blockName="hero">Nobody loves books more than our experts. Browse and filter hundreds of their favorites. Want more suggestions? Check out <a href="#etc">Best Books</a> and <a href="#etc2">125 Books We Love</a>.</BodyText>}
+  foregroundColor="#ffffff"
+  backgroundColor="#bb1d12"
+  backgroundImageSrc="https://placeimg.com/1600/800/arch"
+/>;
+
+export const ExhibitionsHero = () => <Hero
+  heroType={HeroTypes.Primary}
+  heading={<Heading
+    level={1}
+    id={"1"}
+    text={"Syncretic Vibrations: Exploring the Mosaic of Blackness through the Melville J. and Frances S.Herskovits Collection"}
+    blockName={"hero"}
+  />}
+  locationDetails={<Placeholder>Placeholder for locationDetails, which doesn't exist yet</Placeholder>}
+  foregroundColor="#ffffff"
+  backgroundColor="#0071ce" // have these default to section colors
+  backgroundImageSrc="https://images.nypl.org/index.php?id=419627&t=w"
+/>;
+
+export const ResearchNowHero = () => <SectionName><Hero
```

Comment:
Not requesting changes, just some Friday brain-teasers:  
ResearchNow has been renamed to Digital Research Books.  How might we keep something like this up-to-date?

Reply:
Try our best to not give any components specific names.

Reply:
This is in stories, though ... I had been thinking one way Design System can move forward is to build out the "story" version of a component and pass it to the team that's supposed to consume it.  This is also the best solution I can think of to do regression testing (eg: if a component needs to change, how do we make sure it doesn't drastically affect existing users of the component?) But when the product changes, there's not really a good way to signal that back to Design System so that they can update the story ...

Reply:
Yeah, I noticed that after... I guess the person writing the stories should just keep the names up-to-date, and also remove any that are no longer used.

Reply:
Name changes to products don't happen very often. When they do, we should create issues to track down any stories with an old product name and get them updated. I'll do that here.